pallets/msa/src/lib.rs

```diff
@@ -705,6 +738,22 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Adds a list of schema permissions to a delegation relationship.
+	pub fn add_schema_grants_for(
```

Comment:
```suggestion
	pub fn grant_permissions_for_schemas(
```
And also update the comment